doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -231,6 +231,10 @@ Serial LTE modem
   * Full modem FOTA support to the ``#XFOTA`` AT command.
   * An ``auto_connect`` operation in the ``#XCARRIER`` carrier command.
     The operation controls automatic registration of UE to LTE network.
+  * A ``log_data`` operation in the ``#XCARRIER`` carrier command.
+    The operation sends log data using the Event Log object to be read by the LwM2M server.
+  * Support for the Binary App Data Container object as an alternative to the App Data Container object.
+    This can be used through the ``app_data`` operation in the ``#XCARRIER`` carrier command.
```

Comment:
I think we also need an entry for  LwM2M carrier in the "Library section".

* :ref:`liblwm2m_carrier_readme` library:

  * Updated to 3.3.0.
    See the :ref:`liblwm2m_carrier_changelog` for detailed information.